shared/ui-components/src/applications/Client.d.ts

```diff
@@ -0,0 +1,457 @@
+import {
+  OpenAPIClient,
+  Parameters,
+  UnknownParamsObject,
+  OperationResponse,
+  AxiosRequestConfig,
+} from "openapi-client-axios"
+
+declare namespace Components {
```

Comment:
Is this file auto-generated? If not, it seems like a bunch of duplicate definitions, there must be a better way to keep this in sync with `backend/core`?